dcat/index.html

```diff
@@ -2495,7 +2495,7 @@ <h4>Property: in series</h4>
                 <!--tr><th class="prop">Inverse property of:</th>
                 <td><a href="#Property:dataset_series_series_member"><code>dcat:seriesMember</code></a></td>
                 </tr-->
-                <tr><th class="prop">Usage note:</th><td> Normally, child datasets in dataset series are represented as <code>dcat:Dataset</code>. The use of <code>dcat:Distribution</code> for typing child datasets is however recognized as a possible alternative, whenever it addresses more effectively the requirements of a given application scenario.</td></tr>
+                <tr><th class="prop">Usage note:</th><td> Child datasets in dataset series SHOULD be represented as <code>dcat:Dataset</code>.<!--The use of <code>dcat:Distribution</code> for typing child datasets is however recognized as a possible alternative, whenever it addresses more effectively the requirements of a given application scenario.--></td></tr>
```

Comment:
@riccardoAlbertoni , I wonder whether we'd rather drop altogether the usage note. The fact that members of a series are instances of `dcat:Dataset` is already made explicit in the definition of this property, so there's no benefit in restating it in the usage note.

Reply:
I am quite tempted to drop the note.

Though, I wonder if dropping the note might imply that  Items of the Dataset series MUST be Datasets, namely if readers might perceive the message stronger than what I thought we wanted to deliver.

"SHOULD" discourages other solutions, but it accepts exceptions.

Reply:
I see the point, but using SHOULD without indicating alternatives would be close to a MUST. As we have a specific section about alternative solutions, I would just add a link to that section from both `dcat:DatasetSeries` and `dcat:inSeries`.

Reply:
Yes - that is the problem. A SHOULD alongside a MAY is probably needed here.

Reply:
Andrea's suggestion has been implemented via [62945d9](https://github.com/w3c/dxwg/pull/1458/commits/62945d971c2c41f2d8f14c86db9323c6518ebdf7)